utilities/colors.scss

```diff
@@ -1,6 +1,6 @@
 @import '../variables/colors';
 
-@mixin color-modifiers($list: ('primary', 'warn', 'danger', 'safe'), $main-property: 'background', $contrast-property: '', $selector: '', $color-set: $all-colors) {
+@mixin color-modifiers($list: ('info', 'warn', 'danger', 'safe'), $main-property: 'background', $contrast-property: '', $selector: '', $color-set: $all-colors) {
```

Comment:
Hør med @frideska om denne går bra

Reply:
Gjorde om denne for var en vel unødvending løsning for feedbacken uansett. Men når vi først ser på denne så er den mixin muligens litt overflødig. Siden vi har gått over til css variabler benyttes denne bare i badge for å generere disse klassene for bakgrunnsfarge, vi burde kanskje sett over denne og fjernet denne mixinen